plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -1,11 +1,204 @@
 global proc AEmayaUsdProxyShapeTemplate( string $nodeName )
 {
-	editorTemplate -beginScrollLayout;
+    editorTemplate -beginScrollLayout;
 
-		// include/call base class/node attributes
-		AEsurfaceShapeTemplate $nodeName;
+        // Object Display (from AEdagNodeCommon.mel):
+        editorTemplate -beginLayout (uiRes("m_AEshapeTemplate.kObjectDisplay"));
+            editorTemplate -beginNoOptimize;
+                editorTemplate -suppress "template";
+                editorTemplate -suppress "hideOnPlayback";
+                editorTemplate -suppress "displayHWEnvironment";
+                editorTemplate -suppress "ignoreHwShader";
+                editorTemplate -suppress "intermediateObject";
+                editorTemplate -suppress "hardwareFogMultiplier";
+                editorTemplate -addControl "visibility";
+                editorTemplate -label (uiRes("m_AEdagNodeCommon.kLODVisibility")) -addControl "lodVisibility";
```

Comment:
Current Maya has a label for "Visibility" as well, but that tag doesn't exist in Maya 2020.

Reply:
Any reason why you bother reusing a string defined in Maya at all?  Everything else is kind of hard-coded and in English only, but a couple of them will come from Maya and possibly be in another language.  It should work, and the odds of Maya changing (or moving the location of) one of those strings is pretty small, but just wondering why you're even bothering.

Reply:
Because the proxy shape is a Maya node, so I figured we should try and support the localization as much as possible.